security/src/main/java/org/frankframework/lifecycle/servlets/OAuth2Authenticator.java

```diff
@@ -151,26 +154,25 @@ public class OAuth2Authenticator extends AbstractServletAuthenticator {
 
 	/**
 	 * The client ID to use for the OAuth2 provider.
-	 * <p>
-	 * This is required when using the OAuth2 provider.
-	 * </p>
 	 */
 	private @Setter String clientId = null;
 
 	/**
 	 * The client secret to use for the OAuth2 provider.
-	 * <p>
-	 * This is required when using the OAuth2 provider.
-	 * </p>
 	 */
 	private @Setter String clientSecret = null;
 
+	/**
+	 * The AuthAlias which contains the clientId and clientSecret.
+	 */
+	private @Setter String clientAuthAlias = null;
+
+	private ICredentials clientCredentials;
+
 	/**
 	 * The tenant ID to use for the Azure provider.
-	 * <p>
-	 * This is required when using the Azure provider.
-	 * </p>
 	 */
+	@Mandatory
```

Comment:
Kan deze `@Mandatory` zijn? Als deze alleen voor Azure mandatory is maar misschien niet voor andere OAuth2 providers?

Reply:
Ah ja dat klopt!